automation/README.md

```diff
@@ -27,16 +26,30 @@ It may be used both locally and through CI.
 
   It also handles the cleanup of the container and nets (stop,rm).
 
-- run-integration-tests.mounts: Includes mounts to be used by the
-  oVirt CI (STDCI) worker.
+- run-tests.mounts: Includes mounts to be used by the oVirt CI (STDCI) worker.
 
-- run-integration-tests.packages: Includes the packages needed by
-  the oVirt CI (STDCI) worker.
+- run-tests.packages: Includes the packages needed by the oVirt CI (STDCI)
+  worker.
 
 ## Running the Tests
-Assuming *docker* is installed on the host,
-just run:
-`./automation/run-integration-tests.sh`
+Assuming *docker* is installed on the host, just run:
+`./automation/run-tests.sh`
+
+By default, `./automation/run-tests.sh` will run all tests in CI container.
+You may change the test type by using:
+
+ * `TEST_TYPE=lint ./automation/run-tests.sh`: Static analysis of code.
```

Comment:
There is now an example about how to use long-options for run-tests.sh, IMHO it makes sense to add a `--test-type` flag  as well. Fortunately, it is not too complicated.

Reply:
Done.